configure.ac

```diff
@@ -1,5 +1,5 @@
 AC_PREREQ([2.63])
-AC_INIT([eos-chrome-plugin-updater], [1.0.1], [emdev@endlessm.com], [eos-chrome-plugin-updater], [http://www.endlessm.com/])
+AC_INIT([eos-chrome-plugin-updater], [1.0.2], [emdev@endlessm.com], [eos-chrome-plugin-updater], [http://www.endlessm.com/])
```

Comment:
I think you are creating work for yourself by bumping the version number, adding tags, etc. etc. each time you commit a change – we generate the version number based on the git commits since the last tag. If you prefer to do that work, OK, but we don't do it for any other in-house module.

Reply:
Yep, I just thought it was nicer to get a changelog and all that, but maybe I should stop doing it on future updates :).